eth-contracts/contracts/service/DelegateManager.sol

```diff
@@ -445,7 +440,7 @@ contract DelegateManager is RegistryContract {
         uint totalBalanceInStakingPreSlash = stakingContract.totalStakedFor(_slashAddress);
         require(totalBalanceInStakingPreSlash > 0, "Stake required prior to slash");
         require(
-            totalBalanceInStakingPreSlash > _amount,
+            totalBalanceInStakingPreSlash >= _amount,
             "Cannot slash more than total currently staked");
 
```

Comment:
aren't these two require statements `balance > 0` and `balance >= slashAmount` redundant? like the first one is fully encompassed by the second - seems like you could just remove? instead it would make sense to require that `slashAmount > 0` and then require that `balance >= slashAmount`. wdyt? the >= fix is a good catch though

Reply:
yes